src/chat/chat_panel.rs

```diff
@@ -712,40 +711,13 @@ impl WidgetMatchEvent for ChatPanel {
 }
 
 impl ChatPanel {
-    fn jump_to_bottom_actions(&mut self, cx: &mut Cx, actions: &Actions, scope: &mut Scope) {
+    fn jump_to_bottom_actions(&mut self, cx: &mut Cx, actions: &Actions) {
         if let Some(fe) = self.view(id!(jump_to_bottom)).finger_up(actions) {
             if fe.was_tap() {
                 self.scroll_messages_to_bottom(cx);
                 self.redraw(cx);
             }
         }
-
-        let jump_to_bottom = self.view(id!(jump_to_bottom));
-        match self.state {
-            ChatPanelState::Streaming {
-                auto_scroll_pending: true,
-                ..
-            } => {
-                // We avoid to show this button when the list is auto-scrolling upon
-                // receiving a new message. Otherwise, the button flicks.
-                jump_to_bottom.set_visible(false);
```

Comment:
@noxware How this specific logic was translated to the new approach? I can understand everything else but this one seems lost.

Note: This logic seems to be not longer working on `main`, probably because the logic to detect if the user has scrolled up the messages list is fragile. But I wouldn't lost this logic although we would have to fix it later.